pkg/controller/performanceprofile/components/tuned/tuned_test.go

```diff
@@ -186,15 +261,40 @@ var _ = Describe("Tuned", func() {
 						Devices: []performancev2.Device{
 							{
 								InterfaceName: &netDeviceName,
+								DeviceID:      &netDeviceModelID,
 								VendorID:      &netDeviceVendorID,
+							},
+						}}
+					manifest := getTunedManifest(profile)
+					reservedSet, err := cpuset.Parse(string(*profile.Spec.CPU.Reserved))
```

Comment:
same

Reply:
same answer as above (validation web hook will fail creation of profile)